src/rust/src/x509/extensions.rs

```diff
@@ -416,6 +416,133 @@ fn encode_scts(ext: &pyo3::Bound<'_, pyo3::PyAny>) -> CryptographyResult<Vec<u8>
     Ok(asn1::write_single(&result.as_slice())?)
 }
 
+fn encode_naming_authority<'a>(
+    py: pyo3::Python<'_>,
+    ka_str: &'a cryptography_keepalive::KeepAlive<pyo3::pybacked::PyBackedStr>,
+    py_naming_authority: &pyo3::Bound<'a, pyo3::PyAny>,
+) -> Result<extensions::NamingAuthority<'a>, CryptographyError> {
```

Comment:
You can use `CryptographyResult<extensions::NamingAuthority<'a>>` which is marginally shorter

Reply:
Duly noted :-) also switched to `CryptographyResult` for consistency in other introduced functions as well.